jdaviz/utils.py

```diff
@@ -355,6 +355,15 @@ def flux_conversion(spec, values, original_units, target_units):
     return (values * orig_units).to_value(targ_units, equivalencies=eqv)
 
 
+def _convert_surface_brightness_units(data, from_unit, to_unit):
+    try:
+        quantity = data * u.Unit(from_unit)
+        converted_quantity = quantity.to(u.Unit(to_unit))
+        return converted_quantity.value
+    except u.UnitConversionError:
+        raise ValueError(f"Conversion from {from_unit} to {to_unit} is not possible.")
```

Comment:
I think that in this case we need to just pass/return, not raise the error. Currently you cannot create a moment map, because this raises an error that the unit (e.g. `km/s`) can't convert to surface brightness.